.env

```diff
@@ -6,4 +6,4 @@ NODE_URL=https://public-node.testnet.rsk.co
 NODE_MAINNET_URL=https://public-node.rsk.co
 CYPHER_ESTIMATE_FEE_URL=https://api.blockcypher.com/v1/btc/test3
 CYPHER_ESTIMATE_FEE_MAINNET_URL=https://api.blockcypher.com/v1/btc/main
-# API_URL=https://rootstock-testnet.blockscout.com/api
\ No newline at end of file
+API_URL=https://rootstock-testnet.blockscout.com/api
```

Comment:
Question, this is the testnet URL endpoint, does this PR also handle mainnet? Knowing that the Wallet App still needs to handle mainnet?